sim/teams.ts

```diff
@@ -436,11 +460,16 @@ export const Teams = new class Teams {
 		}
 
 		// moves
-		for (let move of set.moves) {
+		for (let i = 0; i < set.moves.length; i++) {
+			let move = set.moves[i];
+			let PPUps = ``;
 			if (move.startsWith(`Hidden Power `) && move.charAt(13) !== '[') {
 				move = `Hidden Power [${move.slice(13)}]`;
 			}
-			out += `- ${move}  \n`;
+			if (set.movePPUps && !isNaN(set.movePPUps[i]) && set.movePPUps[i] < 3) {
+				PPUps = ` (PP Ups: ${set.movePPUps[i]})`;
```

Comment:
```suggestion
			if ((set.movePPUps?.[i] ?? 3) < 3) {
				PPUps = ` (PP Ups: ${set.movePPUps![i]})`;
```
Easy enough to fix the lint issue. :p